docs/data/tree-view/rich-tree-view/expansion/expansion.md

```diff
@@ -33,6 +33,12 @@ Use the `onItemExpansionToggle` prop if you want to react to an item expansion c
 
 {{"demo": "TrackItemExpansionToggle.js"}}
 
+## Limit expansion to icon container
+
+The demo below shows how to trigger the expansion interaction just by clicking on the icon container instead of the whole Tree Item content.
```

Comment:
Having the name of the prop is always nice

```suggestion
You can use the `expansionTrigger` prop to decide if the expansion interaction should be triggered by clicking on the icon container instead of the whole Tree Item content.
```

Or if we want to also show the value:

```suggestion
You can set the `expansionTrigger` prop to `"iconContainer"` to set the expansion interaction on the icon container instead of the whole Tree Item content.
```

Same for `SimpleTreeView` once we agree on a phrasing